### PR TITLE
Style cleanup fixes for setup.py

### PR DESCRIPTION
**Pull request by juztas**

### CODE REVIEW

**Comment by juztas** [comment]
Another one big cleanup which was reported here: https://github.com/dmwm/CRABServer/pull/5017 Lets see ;)

**Comment by mmascher** [comment]
About

```
Line 125 in defineTheBuild W0612 Unused variable '_dirs'
Line 253 in getWebDir W0612 Unused variable '_dirs'
```

should we rename `_dir` to `_` ?

**Comment by ericvaandering** [comment]
I kind of like the convention that a preceding underscore means you don't use the variable, but at least this way you know what the return of the function is.

File: setup.py [hunk 1]
```diff
@@ -1,52 +1,51 @@
 #!/usr/bin/env python2.6
+#pylint: disable=attribute-defined-outside-init
```
Comment (ericvaandering) [inline setup.py]: I would really suggest NOT telling pylint to ignore things. You can ignore pylint, that's fine, but defining things inside __init__ is good practice. If we didn't think it was, we would have set this to globally ignore such things.

**Comment by mmascher** [comment]
@juztas can we close this one?

**Pull request closed without merging**